.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+
+on:
+  pull_request: {}
+  push:
+    branches: [main]
```

Comment:
I don't think we really need to run this workflow on every change, wdyt about limiting this to run only when a schema file actually changes?

Reply:
@pandemicsyn - Agreed. 👍